prototypes/basic/app/views/select_footer_row.html

```diff
@@ -0,0 +1,36 @@
+{% extends "layouts/main.html" %}
+
+{% block head %}
+{{ super() }}
+{% endblock %}
+
+{% block pageTitle %} – {{ serviceName }} – GOV.UK Prototype Kit {% endblock %}
```

Comment:
I know this is the same on other pages, but could we add some default title in here before the `–`? It just looks odd with the leading dash.